noxfile.py

```diff
@@ -12,7 +12,7 @@ def test(session):
         session.run("pytest", "tests/test_configuration.py")
 
 
-@session(python=["3.9"], reuse_venv=True)
+@session(python=["3.8"], reuse_venv=True)
```

Comment:
why test 3.8 specifically?

Reply:
Changed it to the oldest one so we have to update this less frequently